pandas/core/common.py

```diff
@@ -4,17 +4,16 @@
 Note: pandas.core.common is *not* part of the public API.
 """
 
-import collections
-from collections import abc
+from collections import abc, defaultdict
```

Comment:
Slightly unrelated, but this seems better than mixing the two import styles

Reply:
agreed